apps/docs/src/packages/headless.md

```diff
@@ -0,0 +1,58 @@
+---
+outline: [2, 3]
+---
+
+<script lang="ts" setup>
+import packageJson from "../../../../packages/headless/package.json";
+</script>
+
+# @sit-onyx/headless
+
+::: warning Work in progress / Active development
+This library is currently in early / active development.
+:::
+
+{{ packageJson.description }}.
+
+Inspired by [Melt UI](https://melt-ui.com).
+
+## Installation
+
+<!--
+Make sure that this chapter is kept up to date with installation steps in
+packages/headless/README.md file.
```

Comment:
can't we import markdown content? and put those steps in a separate file that we link from packages/headless/README.md and import it right here.?

Reply:
on the other hand, it's not much content so redundance is OK in this case I guess ^^

Reply:
I plan the remove the redundant content in the README.md files and just put a link to the deployed docs once the deployment works